CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Development
```

Comment:
Can you update with the specific version for release here. Since small bug fix, should be 0.3.1.

Can you also update the version in `nextflow.config` too?

Reply:
Will do before the merge! [36739f1](https://github.com/phac-nml/arboratornf/pull/25/commits/36739f163ab8ca8120c909d0c04f971f0766c08b)